plugins/module_utils/dcim.py

```diff
@@ -46,8 +46,8 @@
 
 
 class NautobotDcimModule(NautobotModule):
-    def __init__(self, module, endpoint):
-        super().__init__(module, endpoint)
+    def __init__(self, module, endpoint, client=None, remove_keys=None):
```

Comment:
Do the `__init__` functions need to be defined for these? Seems like they are not doing anything and can just use standard inheritance instead of mixin

Reply:
Nope we don't actually need any of it! It'll be removed on my next push.